tests/distributed/test_dist_graph_store.py

```diff
@@ -148,7 +150,9 @@ def check_dist_emb(g, num_clients, num_nodes, num_edges):
     from dgl.distributed import DistEmbedding
     # Test sparse emb
     try:
-        emb = DistEmbedding(g.number_of_nodes(), 1, 'emb1', emb_init)
+        group_id = os.environ.get('DGL_GROUP_ID', 0)
+        emb_name = f'emb1_{group_id}'
```

Comment:
do we need to change the embedding names or any DistTensor names? who is responsible for adding the group id? the model developer or the framework?

Reply:
if a user doesn't add the group id in the tensor name, will there be conflicts?

Reply:
for now, it's user's responsibility to specify globally unique name or conflict occurs. so does `group_id`. But I will add the `group_id` specify logic into `tools/launch.py` then user don't need to specify manually. it's transparent for users. But user tries to launch clients/servers on their own without `launch.py`, they should be aware of this and takes repsonsibility.